### PR TITLE
Design invite user email template

### PR DESCRIPTION
**Pull request by makindemayowa**
#### What Does This PR Do?
 Design invite user email template
#### Description Of Task To Be Completed
Design invite user email template Pass in necessary details into the template provided and send an email.
Verify email is received by the recipient
#### Any Background Context You Want To Provide?
N/A
#### How can this be manually tested?
Set the below values in your environment 
```
EMAIL_HOST_USER=
EMAIL_HOST=
EMAIL_HOST_PASSWORD=
```
Then create a mutation to send email e.g
```
mutation SendInvite {
  sendEventInvite(input: {
		eventId: "RXZlbnROb2RlOjEy",
		receiverEmail: "email@example.com"
	}) {
    message
  }
}
```

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/161750658
#### Screenshot(s)
Screenshot with local image(since we save images with Django while working locally)
screen shot 2018-11-08 at 3 32 25 pm

Screenshot with valid image url
screen shot 2018-11-08 at 3 33 42 pm

### CODE REVIEW

File: server/graphql_schemas/event/schema.py [hunk 1]
```diff
@@ -236,68 +237,29 @@ def mutate_and_get_payload(cls, root, info, **input):
...
+        msg.content_subtype = 'html'
+        sent = msg.send()
 
         if sent:
             return cls(message="Event invite delivered")
         else:
             raise GraphQLError("Event invite not delivered")
 
 
-class ValidateEventInvite(relay.ClientIDMutation):
```
Comment (andela-sjames) [inline server/graphql_schemas/event/schema.py]: What's going here???

File: server/graphql_schemas/tests/events/test_mutations.py [hunk 2]
```diff
@@ -259,186 +258,3 @@ def test_send_invite_to_self(self):
         self.request.user = self.event_creator.user
         result = self.client.execute(query, context_value=self.request)
         self.assertMatchSnapshot(result)
-
-    def test_validate_invite_link(self):
```
Comment (andela-sjames) [inline server/graphql_schemas/tests/events/test_mutations.py]: HERE???

File: server/graphql_schemas/event/schema.py (hunk 1 above)
Comment (makindemayowa) [reply server/graphql_schemas/event/schema.py]: The new implementation plan I drew out doesn't require validating event invite as the event details are sent directly to the users' email and the user is always directed to the events page on click of the `view event details` button. So the class and its test are no longer required.

Comment (andela-sjames) [reply server/graphql_schemas/event/schema.py]: Sorry, I am getting to this by this time, security can be broken easily from what I can see here, what if someone decides to manually generate the invite? How would you check that it came from you?

**Comment by parseendavid** [comment]
@andela-sjames please review the changes made in accordance with your review.

**Pull request merged**